test/jdk/java/awt/event/KeyEvent/KeyReleasedInAppletTest/KeyReleasedInAppletTest.java

```diff
@@ -1,234 +0,0 @@
-/*
```

Comment:
The "TestApplet.java" near this file is part of the test and can be removed as well.

Reply:
Fixed.